cobalt/umbrel-app.yml

```
@@ -0,0 +1,36 @@
+manifestVersion: 1
+id: cobalt
+name: cobalt
```

Would name it with an uppercase `C` for consistency.
```suggestion
name: Cobalt
```

---

Hey, thanks for the suggestion! I kept it lowercase because cobalt officially uses that styling. 

Umbrel already has several apps that follow their own naming conventions, like autobrr, code-server, nolooking, ntfy, noStrudel, tautulli und usocial. So I think it makes sense to stay consistent with the app's original branding. 

Let me know what you think! 😊

---

True that, lets leave it like this then!